db/playground.sql

```diff
@@ -0,0 +1,9 @@
+\c nc_news_test;
```

Comment:
Might be worth including this file and the .txt version to the .gitignore

Reply:
done